plugins/jvm-abi-gen/src/org/jetbrains/kotlin/jvm/abi/JvmAbiOutputExtension.kt

```diff
@@ -104,41 +111,17 @@ class JvmAbiOutputExtension(
                                 signature: String?,
                                 exceptions: Array<out String>?
                             ): MethodVisitor? {
-                                val info = methodInfo[Method(name, descriptor)]
-                                    ?: return null
+                                val info = methodInfo[Method(name, descriptor)] ?: return null
```

Comment:
This code is really hard to understand, essentially if you don't know method info it is considered non-public and should not be kept for ABI?

Can we extract this ESSENTIAL check into a well-named function or rename the `info` variable or `methodInfo` map?

It should be something like this:

```js
val abiMethodInfo = abiMethodsInfos[Method(name, descriptor)] ?: return null
```

Otherwise reading this code suggests that `methodInfo` contains info on all methods, but it's only public ABI methods.

Reply:
I agree that `methodInfo` could be named better, but afaik we're trying to avoid unnecessary renaming along with the feature implementation. 

I'd be happy to rename `methodInfo` but would wait for @udalov here. I can even open a separate PR with all such refactorings after this one, no problem.

Reply:
I think this is not related to this PR. We can create a separate PR and discuss improvements there, but that is optional, because personally I find this code to be fine as it is now.